website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -0,0 +1,998 @@
+
+    ### Breaking changes and recommendations
+
+    - For users on dbt version 1.5 and lower with dbt Metrics and Snowflake proxy:
+    - **Impact**: Post-deprecation, queries using the proxy _will not_ run.
+    - **Action required:** _Immediate_ migration is necessary. Refer to the [dbt Semantic Layer migration guide](/guides/sl-migration?step=1)
+
+    - For users on dbt version 1.5 and lower using dbt Metrics without Snowflake proxy:
+    - **Impact**: No immediate disruption, but the package will not receive updates or support after deprecation
+    - **Recommendation**: Plan migration to the re-released Semantic Layer for compatibility with dbt version 1.6 and higher.
+
+    ### Engage and support
+
+    - Feedback and community support &mdash; Engage and share feedback with the dbt Labs team and dbt Community slack using channels like [#dbt-cloud-semantic-layer](https://getdbt.slack.com/archives/C046L0VTVR6) and [#dbt-metricflow](https://getdbt.slack.com/archives/C02CCBBBR1D). Or reach out to your dbt Cloud account representative.
+    - Resources for upgrading &mdash; Refer to some additional info and resources to help you upgrade your dbt version:
+    - [Upgrade version in dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud)
+    - [Version migration guides](/docs/dbt-versions/core-upgrade)
+
+  </expandable>
+
+## November 2023
+
+- <expandable alt_header='dbt Explorer updates'>
+
+    There are new quality-of-life improvements in dbt Cloud for email and Slack notifications about your jobs: 
+
+    - You can add external email addresses and send job notifications to them. External emails can be:
+        - Addresses that are outside of your dbt Cloud account
+        - Third-party integration addresses for configuring notifications to services like Microsoft Teams or PagerDuty 
+    - You can configure notifications for multiple Slack channels. Previously, you could only configure one Slack channel. 
+    - Any account admin can now edit slack notifications, not just the person who created them. 
+
+    To learn more, check out [Job notifications](/docs/deploy/job-notifications).
+
+  </expandable>
+
+- <expandable alt_header='Job notifications'>
+
+    There are new quality-of-life improvements in dbt Cloud for email and Slack notifications about your jobs: 
+
+    - You can add external email addresses and send job notifications to them. External emails can be:
+        - Addresses that are outside of your dbt Cloud account
+        - Third-party integration addresses for configuring notifications to services like Microsoft Teams or PagerDuty 
+    - You can configure notifications for multiple Slack channels. Previously, you could only configure one Slack channel. 
+    - Any account admin can now edit slack notifications, not just the person who created them. 
+
+    To learn more, check out [Job notifications](/docs/deploy/job-notifications).
+
+  </expandable>
+
```

Comment:
i think we're missing the fabric support public preview entry: https://github.com/dbt-labs/docs.getdbt.com/pull/5510/files#diff-8b0982fb3ee4baf1666d659ce7d36d58cab5ae9107876a3ffcc58324516ccb9bL1-L12

Reply:
oh nvm public preview isn't included as you said in the description - resolving this!